rpc/lib/client/http_client.go

```diff
@@ -60,12 +61,13 @@ func makeHTTPClient(remoteAddr string) (string, *http.Client) {
 
 //------------------------------------------------------------------------------------
 
-// JSON rpc takes params as a slice
+// JSONRPCClient takes params as a slice
 type JSONRPCClient struct {
 	address string
 	client  *http.Client
 }
 
+// NewJSONRPCClient takes an address and returns a pointer to an instance of JSONRPCClient
```

Comment:
I think this comment is unnecessary as it doesn't document anything new, what do you think?
Or perhaps
```go
// NewJSONRPCClient creates a JSONRPCClient after making an
// underlying HTTPClient connected to the address specified by remote
```

Reply:
`NewJSONRPCClient returns a JSONRPCClient pointed at the given address` is cool.

The go linter picks up on exposed functions without comment - sometimes there's not much to say, so it's ok if the comment doesn't give us too much info.